src/js/components/MaskedInput/doc.js

```diff
@@ -84,6 +84,11 @@ export const themeDoc = {
     type: 'string',
     defaultValue: '18px',
   },
+  'maskedInput.disabled.opacity': {
+    description: 'The opacity when the MaskedInput is disabled.',
+    type: 'number',
+    defaultValue: 0.3,
```

Comment:
According to your base.js definition, the defaultValue should be undefined.

Reply:
it's falling back on global.control.disabled.opacity -- I will update the documentation so the link is clear.

Going to adjust TextInput docs as well since the same is occurring there.